packages/smart-contracts/src/contracts/StorageFeeCollector.sol

```diff
@@ -1,7 +1,7 @@
-pragma solidity ^0.5.0;
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
 
-import "@openzeppelin/contracts/math/SafeMath.sol";
-import "@openzeppelin/contracts/access/roles/WhitelistAdminRole.sol";
+import "./legacy_openzeppelin/contracts/access/roles/WhitelistAdminRole.sol";
```

Comment:
Have you considered using [Roles](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/be5ed7364b93daccbb74a09e3f5ec1be6c458097/contracts/access/Roles.sol) ?

Reply:
Yes, It was my previous PR. I dashback on it change the ABI's.